docker/main/schedulerservice/Dockerfile

```diff
@@ -17,6 +17,7 @@ COPY --from=sources /DIST /DIST
 #COPY --chown=mpi:mpi conf/redis.conf /usr/local/etc/redis/redis.conf
 #COPY --chown=mpi:mpi dmod/scheduler/image_and_domain.yaml /nwm/
 COPY ./image_and_domain.yaml /nwm/
+COPY ./resources.yml /code/
```

Comment:
I don't think we want the configuration of available resources for the runtime environment statically built into the Docker image.  I could be convinced otherwise, but initially it seems like this should be something more dynamically adjustable (e.g., an external file brought into the container via a host volume mount).

Reply:
Made both resources and image_and_domain mounts in the deployment.